_data/projects/bbox-visualizer.yml

```
@@ -0,0 +1,11 @@
+name: bbox-visualizer
+desc: An easy-to-use tool for drawing and labeling bounding boxes after detection and recognition.
+site: https://github.com/shoumikchow/bbox-visualizer
+tags:
+  - python
+  - computer-vision
+  - object-detection
+  - object-recognition
+upforgrabs:
+  name: bbox-visualizer
```

This is the item that the automation has issues with:

```suggestion
  name: "good first issue"
```

---

Thanks!